vagrant/provisioning/roles/privacy/tasks/main.yml

```diff
@@ -84,6 +84,7 @@
     force: no
     src: application.properties
     dest: "{{ root_folder }}/data/arkcase-home/.external-portal-api"
+  when: arkcase_version != "" and arkcase_version is version('2020.14', '<')
```

Comment:
Out of curiosity, I'm guessing that version e.g. '3.3.9' evaluates as less than '2020.14'?

Reply:
Yes. We were lucky there :).